packages/design-system/src/components/Navigations/MenuBar/MenuBar.tsx

```diff
@@ -0,0 +1,113 @@
+import { useCallback, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { colors } from '../../../styles/colors';
+import { Icon } from '../../Base/Icon';
+import { VStack } from '../../Layout/StackContainer';
+
+const Component = styled.div`
+  padding: 30px 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 64px;
+  height: 100vh;
+  background-color: ${colors.grey12};
+  position: absolute;
```

Comment:
absoulte는 부모 컨테이너가 기준이라서 뷰포트 기준으로 바뀌어야 할 것 같습니다.
```suggestion
  position: fixed;
```